content/en/docs/community_roles/contributor.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Project Contributor
+hide_readingtime: true
+weight: 20
+---
+
+A Project Contributor plays a pivotal role in shaping the project through code, documentation, or design. Whether fixing bugs, implementing features, or enhancing the user experience, contributors bring fresh perspectives and energy. Their work supports the project’s evolution, bridging the gap between the project's goals and the community's needs.
+
+## How to Become a Contributor
+
+To contribute to a project, individuals must first interact with _Paul Botsco_ to sign the Contributor License Agreement (CLA), ensuring that their contributions comply with legal requirements.
+
+_Paul Botsco_ will comment on your first pull request, providing you with the latest links and helpful information to sign the CLA.
+
+Once the CLA is signed, contributors can engage with the project by opening issues to report bugs, request features, or suggest improvements. They can also submit pull requests (PRs) to offer code, documentation, or design enhancements.
+
+In the rare event that the CLA text is updated, Paul Botsco will comment on your next pull request to guide you through the process again.
```

Comment:
Should we include a cross reference to https://contribute.sonatype.com/docs/contributing/ here?